ocaml/idl/datamodel.ml

```diff
@@ -1200,11 +1200,20 @@ let _ =
     ~doc:"The requested update could not be found. Please upload the update again. This can occur when you run xe update-pool-clean before xe update-apply. " ();
   error Api_errors.update_pool_apply_failed [ "hosts" ]
     ~doc:"The update cannot be applied for the following host(s)." ();
+  error Api_errors.update_apply_failed [ "output" ]
+    ~doc:"The update apply failed. Please see attached output." ();
   error Api_errors.update_already_applied [ "update" ]
     ~doc:"This update has already been applied." ();
   error Api_errors.update_already_applied_in_pool [ "update" ]
     ~doc:"This update has already been applied to all hosts in the pool." ();
-
+  error Api_errors.update_precheck_failed_unknown_error [ "update"; "info" ]
+    ~doc:"The update precheck stage failed with an unknown error. See attached info for more details." ();
```

Comment:
I think `See attached info for more details` is too casual and doesn't add much value as it is quite natural to inspect other output for clues about a problem. I'd rather re-phrase it as `See the output for more details` or drop it entirely.